selectors/storage.ts

```diff
@@ -1,31 +1,35 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import {Dictionary} from 'mattermost-redux/types/utilities';
+
 import {getPrefix} from 'utils/storage_utils';
 
-export const getGlobalItem = (state, name, defaultValue) => {
+import type {GlobalState} from 'types/store';
+
+export const getGlobalItem = (state: GlobalState, name: string, defaultValue: any) => {
```

Comment:
I don't know if it's worth changing this now since I'm hoping to rewrite all this, but we could possibly parameterize these functions so that the user can override the return type and keep TypeScript happy.
```suggestion
export const getGlobalItem = <T = any>(state: GlobalState, name: string, defaultValue: T) => {
```

Reply:
Done